settings.js

```diff
@@ -58,6 +58,7 @@ window.spinnakerSettings = {
   },
   authEnabled: process.env.AUTH === 'enabled',
   gitSources: ['stash', 'github'],
+  triggerTypes: ['git', 'pipeline', 'docker', 'cron', 'jenkins'],
```

Comment:
so technically not just docker like the issue implies, but allows you to turn on and off which trigger types you want to set